src/screens/MainApp/DiagnoseRequest/screens/DescriptionRequest/index.js

```diff
@@ -75,31 +61,51 @@ const DescriptionRequest = ({ navigation }) => {
 
   return (
     <View style={styles.container}>
-      <Text fontVariant='h2' colorVariant='black' style={styles.title}>
-        Describí las particularidades que notás en tu planta
-      </Text>
-      <TextInput
-        onChangeText={handleInput}
-        value={description}
-        style={styles.input}
-        editable={inputEnabled}
-      />
-      <Text fontVariant='description' colorVariant='black' style={styles.description}>
-        Cuanta mas información escribas, mejor. Por favor, no incluyas datos de contacto.
-      </Text>
-      <Button
-        variant={uploadEnabled ? 'black' : 'gray'}
-        style={styles.button}
-        onPress={handleUpload}
-        disabled={!uploadEnabled}
-      >
-        <ProgressBar progress={uploadProgress} />
-        <Text style={styles.buttonText}>
-          {((uploading) && 'Enviando...') || 'Enviar'}
-        </Text>
-      </Button>
+      <ScrollView contentContainerStyle={styles.scroll}>
+        <Title black style={styles.title}>
+          Describí las particularidades de tu planta
+        </Title>
+        <Description gray style={styles.description}>
+          Cuanta mas información escribas, mejor. Por favor, no incluyas datos de contacto.
+        </Description>
+        <BoxShadow
+          setting={{
+            width: scale(322),
+            height: verticalScale(90),
+            border: 5,
+            opacity: 0.02,
+            radius: 5,
+            style: styles.shadow
```

Comment:
Use scalingUnits for border and radius too. I would change it to `moderateScale(border)`

Reply:
@jmbidev